python/triton/code_gen.py

```diff
@@ -507,6 +511,7 @@ def __init__(self, device: int, bin: Binary):
                                                       device)
         self.bin = bin
         self.asm = bin.asm
+        self.sass = None
```

Comment:
Could we have `self.sass` be exposed as `self.asm['sass']` ?

Reply:
Yes!